{{ cookiecutter.repo_name }}/gradle/dependencies.gradle

```diff
@@ -14,12 +14,12 @@ ext.buildConfig         = buildConfig
 
 def versions            = [:]
 versions.kotlin         = '1.2.71'
-versions.supportLibrary = '28.0.0'
+versions.supportLibrary = '1.0.0'
 versions.arch           = '1.1.1'
-versions.archRoom       = '1.1.1'
-versions.atsl           = '1.0.2'
-versions.espresso       = '3.0.2'
-versions.dagger         = '2.17'
+versions.archRoom       = '2.0.0'
+versions.atsl           = '1.1.0-alpha3'
+versions.espresso       = '3.1.0-alpha3'
+versions.dagger         = '2.16'
```

Comment:
any reason to downgrade `dagger`?

Reply:
Yes, there is currently a known issue with AndroidX and Dagger 2.17, documented here:
https://github.com/google/dagger/issues/1245
https://issuetracker.google.com/issues/115738511

We will update Dagger once the issue is resolved.

Reply:
I see. Thanks.